pegasus/migrations/75_add_created_at_column_to_storage_apps.rb

```diff
@@ -0,0 +1,41 @@
+# Migration for extracting JSON.parse(row[:value])['createdAt'] into a :created_at column.
+Sequel.migration do
+  up do
+    add_column :storage_apps, :created_at, DateTime
+
+    batch_update do |row|
+      value = JSON.parse(row[:value])
+      created = Time.parse(value['createdAt']) if value['createdAt']
+
+      # Remove metadata keys from :value hash (they're stored in separate columns or calculated dynamically).
+      %w(id isOwner createdAt updatedAt).each{ |key| value.delete(key) }
```

Comment:
To make this change less scary we could wait to remove the extra metadata until we verify there are no problems with the new column.